scene/gui/tree.h

```diff
@@ -724,8 +724,9 @@ class Tree : public Control {
 
 	int get_item_offset(TreeItem *p_item) const;
 	Rect2 get_item_rect(TreeItem *p_item, int p_column = -1, int p_button = -1) const;
-	bool edit_selected();
+	bool edit_selected(bool p_force_edit);
```

Comment:
```suggestion
	bool edit_selected(bool p_force_edit = false);
```
Instead of changing the existing calls.

Reply:
Made the change.